ui/webui/src/components/AnacondaAboutModal.jsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2022 Red Hat, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with This program; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React, { useState, useEffect } from "react";
+import {
+    AboutModal,
+    Button,
+    Flex,
+    TextContent,
+    TextList,
+    TextListItem,
+} from "@patternfly/react-core";
+import { ExternalLinkAltIcon } from "@patternfly/react-icons";
+import { getAnacondaVersion } from "../helpers/product";
+import "../components/app.scss";
+
+export const AnacondaAboutModal = ({ isModalOpen, setIsAboutModalOpen }) => {
+    const toggleModal = () => {
+        setIsAboutModalOpen(!isModalOpen);
+    };
+
+    const [anacondaVersion, setAnacondaVersion] = useState("");
+
+    useEffect(() => {
+        getAnacondaVersion().then(content =>
+            setAnacondaVersion(content.split(" ").slice(-1)[0].replace("\n", ""))
+        );
+    }, []);
+
+    return (
+        <>
+            <AboutModal
+              style={{ marginTop: "0" }}
+              isOpen={isModalOpen}
+              onClose={toggleModal}
+              trademark="© 2023 Red Hat, Inc."
+              productName="Red Hat Enterprise Linux 9 installer"
+            >
+                <TextContent className="pf-u-py-xl" style={{ marginBottom: "48px" }}>
+                    <h1>Powered by Anaconda</h1>
```

Comment:
This is most definitely _not_ an H1.

As the about part gets translated to an H1, this really should be an H2.

From: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#usage_notes

> Do not skip heading levels: always start from `<h1>`, followed by `<h2>` and so on.

> Avoid using multiple `<h1>` elements on one page